src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp

```diff
@@ -548,8 +548,8 @@ INSTANTIATE_TEST_SUITE_P(nightly_FC_3D_BF16, MatMulLayerCPUTest, testParams3DBF1
 
 const std::vector<ShapeRelatedParams> IS2D_Brgemm_smoke = {
     // needed by 'IS2D_Brgconv1x1_smoke'
-    {static_shapes_to_test_representation({{29, 120}, {120, 120}}), {true, false}},
-    {static_shapes_to_test_representation({{39, 120}, {120, 120}}), {true, false}},
+    {static_shapes_to_test_representation({{1, 120}, {120, 120}}), {true, false}},
+    {static_shapes_to_test_representation({{1, 128}, {128, 166}}), {true, false}},
```

Comment:
Why did you decide to change the K dimension here?

Reply:
`{1, 128}, {128, 166}` will be used to test switch: 'inner product->1x1->inner product->1x1' in https://github.com/openvinotoolkit/openvino/blob/0be83494d04cbfb8adcd6d092588415961dbcfab/src/plugins/intel_cpu/tests/functional/single_layer_tests/matmul.cpp#L636-L638

Adding `{1, 128}, {128, 166}` here will be sure it should use inner product kernel and our switch test will really change from one kernel to another.

Reply:
Yes I see, but I asked about K and N dimmensions, 120 -> 128, 120 -> 166? What is the idea behind this change?

Reply:
No special reason and it's just a new different test case.